taprpc/taprpc_utils.go

```diff
@@ -1,6 +1,17 @@
 package taprpc
 
-import "google.golang.org/protobuf/encoding/protojson"
+import (
+	"encoding/json"
+	"fmt"
+
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+const (
+	// MetadataDecDisplayKey is the JSON key used in the metadata field of a
+	// minted asset to express the decimal display of the minted asset.
+	MetadataDecDisplayKey = "tap_asset_decimal_display"
```

Comment:
I don't think we need the `tap_asset_` prefix in the key.

Reply:
This was only done to minimize risk of a random conflict with a user's keys

Since we now error out on conflict I'll simplify this key.